submissions/sayopaul/Week1/fizz.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+	for($i=1; $i<=100; $i++){
+
+		$output =  ($i % 15 === 0) ? "FizzBuzz".PHP_EOL  :  ( ($i % 3 === 0)  ? "Fizz".PHP_EOL :  ( ($i % 5 === 0 ) ? "Buzz".PHP_EOL : $i.PHP_EOL ));
+
+		/*
+			original version(longer)
+		if($i % 15 === 0 ){
+
+			$output = "FizzBuzz".PHP_EOL;	
+
+			
```

Comment:
Too much new lines added, please make it more concise and remove the unnecessary new lines.